js/src/lib/components/InferenceWidget/shared/WidgetHeader/WidgetHeader.svelte

```diff
@@ -22,7 +28,14 @@
 	class="flex items-center justify-between flex-wrap w-full max-w-full text-sm text-gray-500 mb-0.5"
 >
 	{#if pipeline}
-		<ModelPipelineTag classNames="mr-2 mb-1.5" {pipeline} />
+		<a
+			class={TASKS_DATA[task] ? "hover:underline" : undefined}
```

Comment:
the `hover:underline` doesn't seem to have any effect on firefox, but it's working on chrome, strange (no need to spend too much time on this though)

Reply:
@Pierrci  I'll merge first (as some people are eager to get this PR out) then investigate :)